programmers/모의고사.js

```diff
@@ -0,0 +1,29 @@
+function solution(answers) {
+   const numberOne = [1,2,3,4,5];
+   const numberTwo = [2,1,2,3,2,4,2,5];
+   const numberThree = [3,3,1,1,2,2,4,4,5,5];
+   let firstCnt = 0;
+   let secondCnt = 0;
+   let thirdCnt = 0;
+
+   for(let i = 0; i < answers.length; i++) {
+     if (numberOne[i % 5] === answers[i]) firstCnt++;
+     if (numberTwo[i % 8] === answers[i]) secondCnt++;
+     if (numberThree[i % 10] === answers[i]) thirdCnt++; 
+   }
```

Comment:
5, 8, 10 이렇게 보다 patterns[0].length 이렇게 넣는게 좋아 보임
```suggestion
   for (let i = 0; i < answers.length; i++) {
     if (patterns[0][i % patterns[0].length] === answers[i]) count[0]++;
     if (patterns[1][i % patterns[1].length] === answers[i]) count[1]++;
     if (patterns[2][i % patterns[2].length] === answers[i]) count[2]++; 
   }
```